src/bytesToUuid.js

```diff
@@ -14,28 +14,28 @@ function bytesToUuid(buf, offset) {
   const bth = byteToHex;
 
   // join used to fix memory issue caused by concatenation: https://bugs.chromium.org/p/v8/issues/detail?id=3175#c4
```

Comment:
The issue linked above is still unresolved.

@awwit have you tried the examples in that issue in recent Node.js or Chrome versions to see if the issue still exists?

@broofa can you shed some light on this? I saw the discussion in https://github.com/uuidjs/uuid/pull/267 but cannot judge if this is still an issue?

Reply:
Oh, I just saw https://github.com/uuidjs/uuid/pull/267#issuecomment-378291582 which I must have missed while reading the other issue initially.

Seems like `.toLowerCase()` is also a workaround for the memory issue? If this turns out to be true (@broofa / @sava-smith can you confirm) then we should at least update the comment here to reflect the new hack?

Reply:
@ctavan I tried different ways to fix this bug. It still exists in the V8 engine. (Did not check for other engines).

The fastest option was ".toLowerCase()" on V8 engine. It also forms a normal-sized string like `.join('')`.

There is still a compromise option `(' ' + str).slice(1)` which works even faster, but consumes a little more memory for the result string.